### DISCUSSION TITLE
Overlay camera support for positioning

### DISCUSSION THREAD

**Discussion by duan-dt** [arkypita/LaserGRBL#2384] [Ideas]
Hi

Great program, thanks for making this free and accessible to the millions of people just doing engraving and cutting as a hobby or small business. 

One feature I enjoyed in Lightburn was the camera overlay feature which gave me the ability to position my engraving and cuts precisely Are there any plans for this feature in the near future? It is really the only thing preventing me from making the switch completely since the Creality Falcon 2 Pro does not have any other way of ensuring the positioning is correct.

**Comment by MArTI9999** [comment]
There's a video on YouTube showing how to use the OV7670 camera module with an arduino. I think this technical solution could be added  to LaserGRBL by adding the option to connect a second port for the arduino that captures the image from the camera and then displays this image of the workbench (not necessarily live) on the screen.Then you could use a mouse to drag your image for engrave to aligne with material and scale it live.
Of course there would be need to enable homing cycle and some kind of stand for camera or attachment to cover of the machine for proper working.
Unfortunately, I don't have any programming skills to even attempt to do this. So if my idea is somewhat what you need either you do it yourself or we need to wait until creator or someone else make something similar for us.

https://www.youtube.com/watch?v=R94WZH8XAvM
